### PR TITLE
Upgrade vendor list performance

### PR DESCRIPTION
**Pull request by osk-8**
Extract records from vendors list to `Vendor` component and set is as a component which cannot be rerender.

### CODE REVIEW

File: src/components/popup/details/vendors/vendors.jsx [hunk 1]
```diff
@@ -94,35 +96,14 @@ export default class Vendors extends Component {
 				<div class={style.vendorContent}>
 					<table class={style.vendorList}>
 						<tbody>
-						{vendors.map(({ id, name, policyUrl, purposeIds, legIntPurposeIds, featureIds }, index) => (
+						{vendors.map(({ id, name, policyUrl, purposeIds=[], legIntPurposeIds=[], featureIds=[] }, index) => (
```
Comment (mvciekzvbek) [inline src/components/popup/details/vendors/vendors.jsx:122]: Is it necessary? I thought our vendor list is already validated by our backend systems (properties like purposeIds, legIntPurposIds and featureIds are empty arrays if they does not exist)

File: src/components/popup/details/vendors/vendor.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import Label from "../../../label/label";
+
+class LocalLabel extends Label {
+	static defaultProps = {
+		prefix: 'vendors'
+	};
+}
+
+export default class Vendor extends Component {
+	shouldComponentUpdate() {
```
Comment (mvciekzvbek) [inline src/components/popup/details/vendors/vendor.jsx:14]: Maybe it is worth to mention that there is a possibility to check if new props are different than current - https://reactjs.org/docs/optimizing-performance.html#shouldcomponentupdate-in-action. Did you consider using it?

**Review by mvciekzvbek — APPROVED** [review]

File: src/components/popup/details/vendors/vendors.jsx (hunk 1 above)
Comment (osk-8) [reply src/components/popup/details/vendors/vendors.jsx:122]: No, but at no cost we secure ourselves against the edge case in which these arrays don't reach our component

File: src/components/popup/details/vendors/vendor.jsx (hunk 2 above)
Comment (osk-8) [reply src/components/popup/details/vendors/vendor.jsx:14]: Yes, but I wonder in which case these attributes can be updated after first render. Do you have any?

**Pull request merged**